aeron-client/src/main/java/io/aeron/reports/LossReport.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.aeron.driver.reports;
+package io.aeron.reports;
```

Comment:
Should be a sub-package of `io.aeron.tooling`. The latter probably should be named `io.aeron.tools`.